combirepo/__main__.py

```diff
@@ -31,7 +31,7 @@ def main(args=None):
     config_parser = ConfigParser()
     if os.path.isfile(config_parser.path):
         config_parameters = config_parser.parse()
-        parameters = config_parameters + commandline_parameters
+        parameters = commandline_parameters + config_parameters
```

Comment:
Why is this done? Doesn't it change priority?

Reply:
> It is said, that commandline parameters are preferred to config file, but in
fact it was exactly opposite.

Reply:
@rayslava is right, current implementation gives priority to the second argument of __add__ operator. See https://github.com/Samsung/combirepo/blob/f06168f58ba1d860eb276e4426fd9e5ca5e05def/combirepo/parameters.py#L311

Reply:
So the patch seems to be erroneous, because command line arguments go as second argument and thus are preferred.

Sorry for such inconvenient interface :)

Reply:
(see comment and code of __add__ method)

Reply:
> @rayslava is right, current implementation gives priority to the second argument of __add__ operator.

It gives priority to the first argument.

Reply:
Ok, verified and reverted.

Reply:
OK